syntax/nirum.vim

```diff
@@ -19,7 +19,7 @@ syn keyword NirumPrimitiveNumberType bigint decimal int32 int64 float32 float64
 syn keyword NirumPrimitiveStringType text binary
 syn keyword NirumPrimitiveTimeType datetime date
 syn keyword NirumPrimitiveBoolType bool
-syn keyword NirumPrimitiveEtcType uuid uri
+syn keyword NirumPrimitiveEtcType uuid url
```

Comment:
Until the minor version of the Nirum compiler (0.4.0), both names coexist.  Could we highlight deprecated or obsolete keywords in a different way from others?  E.g., red background.